docs/static/releasenotes.asciidoc

```diff
@@ -1,17 +1,25 @@
 [[releasenotes]]
 == Release Notes
 
+coming[8.0.0]
+
 This section summarizes the changes in the following releases:
 
-* <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
-* <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 
-[[logstash-7-0-0-alpha1]]
-=== Logstash 7.0.0-alpha1 Release Notes
+//* <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
```

Comment:
```suggestion
* <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
```

Reply:
I don't want links to be live.  No point inviting a user to click a link only to find out that there's nothing there. 
Even though they are commented out, it's still a good idea to get the 8.0 ref in there. Thanks!